Content.Shared/CCVar/CCVars.cs

```diff
@@ -511,6 +511,10 @@ public sealed class CCVars : CVars
 
         public static readonly CVarDef<bool> AdminSoundsEnabled =
             CVarDef.Create("audio.admin_sounds_enabled", true, CVar.ARCHIVE | CVar.CLIENTONLY);
+        public static readonly CVarDef<string> AdminChatSoundPath =
+            CVarDef.Create("audio.admin_chat_sound_path", "/Audio/Items/pop.ogg", CVar.ARCHIVE | CVar.SERVERONLY);
```

Comment:
It may be better to make this and the volume cvar a replicated client cvar, and then individually send the admin message to each admin with their requested notification noise (if any)